src/components/ContextualSaveBar/ContextualSaveBar.tsx

```diff
@@ -20,19 +20,19 @@ class ContextualSaveBar extends React.PureComponent<ComposedProps, never> {
     const {
       props: {polaris, context, ...rest},
     } = this;
-    context.frame.setContextualSaveBar(rest);
```

Comment:
Removing the `frame` namespace, similar to `polaris`

Reply:
Is it useful to have this namespaced to make it clear what component these methods relate to?

Reply:
I'll be hookifying the ~5 components that use `withContext` so they'll have a namespace after that
```
const frame = React.useContext(FrameContext);
frame.setContextualSaveBar(rest);
```